code/obj/flock/cage.dm

```diff
@@ -195,6 +195,7 @@
 				eating_occupant = 0
 				playsound(src, "sound/weapons/nano-blade-1.ogg", 50, 1)
 				if(occupant)
+					src?.flock?.updateEnemy(occupant)
```

Comment:
I think this should be moved out, it means that the enemy will only be updated if it has something that can be eaten?